src/protocols/secure_channel/PASESession.h

```diff
@@ -276,13 +276,24 @@ class DLL_EXPORT PASESession : public Messaging::ExchangeDelegate, public Pairin
 };
 
 /*
- * The following constants are node IDs that test devices and test
- * controllers use while using the SecurePairingUsingTestSecret to
- * establish secure channel
+ * The following constants are node IDs and CASE Authenticated Tags (CATs)
+ * that test devices and test controllers use while using the
+ * SecurePairingUsingTestSecret to establish secure channel.
  */
 constexpr chip::NodeId kTestControllerNodeId = 112233;
 constexpr chip::NodeId kTestDeviceNodeId     = 12344321;
```

Comment:
This file isn't really a good place for kTestControllerNodeId and kTestDeviceNodeId in the first place.  So adding all of the other test values here seems like we are just going further down the wrong path.  Shouldn't these be in a separate file?

None of these test values would ever be needed in a production build.

Reply:
removed